test/utils/fixture/fixture.go

```diff
@@ -94,13 +96,63 @@ func SetupWithOptions(name string, nrtList nrtv1alpha2.NodeResourceTopologyList,
 	}
 	randomizeName := (options & OptionRandomizeName) == OptionRandomizeName
 	avoidCooldown := (options & OptionAvoidCooldown) == OptionAvoidCooldown
+	isSuiteSetup := (options & OptionIsSuite) == OptionIsSuite
+	ginkgo.By("set up the test namespace")
 	ns, err := setupNamespace(e2eclient.Client, name, randomizeName)
 	if err != nil {
 		klog.Errorf("cannot setup namespace %q: %v", name, err)
 		return nil, err
 	}
-	ginkgo.By(fmt.Sprintf("set up the test namespace %q", ns.Name))
+	klog.Infof("test namespace %q was set up successfully", ns.Name)
```

Comment:
this should be logged the same way as "set up the test namespace", so this should be a `ginkgo.By`

Reply:
why? I understand ginkgo.By usually used to highlight the next step to be executed. Since I moved By to line 100 to declare that the test is about to set up the ns, when the setup is done, I log the info pointing out the ns name. I did the same for NRT data.
I usually like to use ginkgo.By to highlight milestones/main steps of the test/function and klog for additional details while running the step's code

Reply:
for consistency. We notify the begin of an action using `ginkgo.By`, which is fine and I agree with your way of using it, but we notify the end of the same action using `klog.Infof`.
Not a big deal though